pgmpy/estimators/HillClimbSearch.py

```diff
@@ -158,6 +158,8 @@ def estimate(self, start=None, tabu_length=0, max_indegree=None):
 
             if best_operation is None:
                 break
+            if best_score_delta < epsilon:
```

Comment:
Rather than doing this it's better that we have
eps = 1e-5
if prev_score_delta - score_delta < eps
.... then break